README.md

```diff
@@ -22,16 +22,49 @@ Specify a different UID/GID mapping:
 $ fuse-overlayfs -o uidmapping=0:10:100:100:10000:2000,gidmapping=0:10:100:100:10000:2000,lowerdir=lowerdir/a:lowerdir/b,upperdir=up,workdir=workdir merged
 ```
 
-Build Requirements:
+Requirements:
 =======================================================
 
-fuse-overlayfs requires libfuse > v3.2.1
+Please note that `fuse-overlayfs` requires Linux Kernel > v4.8.0.
```

Comment:
why is Linux v4.8.0 required?

Reply:
I faced this problem myself at the time of first install, and finally spotted [your comment](https://github.com/containers/libpod/issues/3109#issuecomment-491848145) that clarifies this.

In addition, there is a typo in my code, which I now fixed!

Reply:
I faced this problem myself at the time of first install, and finally spotted [your comment](https://github.com/containers/libpod/issues/3109#issuecomment-491848145) that clarifies this.

In addition, there is a typo in my code, which I now fixed!